### PR TITLE
bootstrap style cards added to foods page and image added as a hardco…

### LINKED ISSUE #4: Make foods menu UI better
I want a UI made using bootstrap cards, three cards in one line, each card representing one food with order and add to card button.

### PR DESCRIPTION
**Pull request by henriquepesi**
Fixes #4 
Styling added to foods page with bootstrap cards

### CODE REVIEW

**Review by rahgurung — CHANGES_REQUESTED** [review]
Looks like your indentations don't match with the current code, can you make them same for readability ?

**Comment by rahgurung** [comment]
@henriquepesi Thanks for the PR though, it looks great. But it is not mobile responsive, can you make it?

**Comment by henriquepesi** [comment]
hi @gurungrahul2 Yes of course! I'm gonna correct the indentation and make the responsive by the end of the day

File: application/views/foods/index.php [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-  <h4 style="padding-top:15px;"> <?php echo $food['name']; ?> -
-  <small><?php echo $food['price']; ?> </small>
-  - <?php echo $rnames[$key]; ?>
-  <a class="btn btn-dark food-page" role="button" href="/foods/order_food/<?php echo $food['id']  ?>"> Order </a>
-  <a class="btn btn-success food-page" role="button" href="/foods/add_to_cart/<?php echo $food['id']  ?>"> Add to Cart </a>
- </h4>
-  <br><hr>
-<?php endforeach; ?>
+<div class="row">
+	<?php foreach($foods as $key => $food) : ?>
```
Comment (rahgurung) [inline application/views/foods/index.php]: @henriquepesi Please use, 2 spaces as indentation.

**Review by rahgurung — COMMENTED** [review]
Responsiveness works fine, can you fix the below.

**Comment by henriquepesi** [comment]
@gurungrahul2 I think everything is fine now, can you check it out?

File: system/libraries/javascript/Jquery.php [hunk 2]
```diff
@@ -1,40 +1,110 @@
-<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed');
-
+<?php
```
Comment (rahgurung) [inline system/libraries/javascript/Jquery.php:1]: @henriquepesi Can you tell me why are you touching files in system/* ?

**Review by rahgurung — CHANGES_REQUESTED** [review]
Please don't change files in system/* and touch only the files for food pages.

File: system/libraries/javascript/Jquery.php (hunk 2 above)
Comment (henriquepesi) [reply system/libraries/javascript/Jquery.php:1]: @gurungrahul2 sorry, I think I've made a mistake with some  files, I've sent a new PR, please ignore this PR above

Comment (rahgurung) [reply system/libraries/javascript/Jquery.php:1]: @henriquepesi I can't find your new PR.

**Comment by rahgurung** [comment]
@henriquepesi still there are unwanted stuff in this PR

**Comment by rahgurung** [comment]
#10 I used your changes and made a new PR, thanks for helping me out :)

**Comment by henriquepesi** [comment]
> #10 I used your changes and made a new PR, thanks for helping me out :)

That's awesome, I'm glad that everything worked :)

**Pull request closed without merging**